x-pack/test/functional/apps/maps/discover.js

```diff
@@ -9,12 +9,24 @@ import expect from '@kbn/expect';
 export default function ({ getService, getPageObjects }) {
   const queryBar = getService('queryBar');
   const PageObjects = getPageObjects(['common', 'discover', 'header', 'maps', 'timePicker']);
+  const security = getService('security');
 
   describe('discover visualize button', () => {
     beforeEach(async () => {
+      await security.testUser.setRoles([
+        'test_logstash_reader',
+        'global_maps_all',
+        'geoshape_data_reader',
+        'global_discover_read',
+        'global_visualize_read',
+      ]);
```

Comment:
I noticed the other setRoles added in this PR used the `false` second parameter so that it doesn't refresh the browser.  That could probably also be added here since the next step it to navigate (which always refreshes the browser as well).  But I don't think there's any harm leaving this as-is.

Reply:
@LeeDr yes visualize_create_menu.js wouln't pass without refreshing the browser. Thanks!